examples/data_labeler/data_labeler_predict_train.ipynb

```diff
@@ -0,0 +1,370 @@
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "legendary-kitty",
+   "metadata": {},
+   "source": [
+    "In this example, we utilize the data labeler component of the Data Profiler to detect the sensitive information for both structured and unstructured data. Data Profiler can be transfer learned with the new datasets to get better results on some specific domains.\n",
+    "\n",
+    "First, let's import the libraries needed for this example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "matched-scoop",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.insert(0, '../..')\n",
+    "import os\n",
+    "import dataprofiler as dp\n",
+    "import json\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "negative-sailing",
+   "metadata": {},
+   "source": [
+    "## Structured data"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "julian-retention",
+   "metadata": {},
+   "source": [
+    "We use the aws honeypot dataset in the test folder for this example. First, look at the data using data reader class of the data profiler. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "american-albania",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = dp.Data(\"../../dataprofiler/tests/data/csv/aws_honeypot_marx_geo.csv\")\n",
+    "df_data = data.data\n",
+    "df_data.head()"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "pharmaceutical-chinese",
+   "metadata": {},
+   "source": [
+    "The data contains 16 columns, each of which has different data label. Next, we will use data labeler of the data profiler to predict the label for each column in this tabular dataset. To use only the labeling functionality, other options are disabled."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "colored-outside",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set option to run only data labeler\n",
+    "profile_options = dp.ProfilerOptions()\n",
+    "profile_options.set({\"text.is_enabled\": False, \n",
+    "                     \"int.is_enabled\": False, \n",
+    "                     \"float.is_enabled\": False, \n",
+    "                     \"order.is_enabled\": False, \n",
+    "                     \"category.is_enabled\": False, \n",
+    "                     \"datetime.is_enabled\": False,})\n",
+    "\n",
+    "profile = dp.Profiler(data, profiler_options=profile_options)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "agreed-benefit",
+   "metadata": {},
+   "source": [
+    "The results are shown below:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "religious-absolute",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get the prediction from data profiler\n",
+    "def get_structured_results(results):\n",
```

Comment:
Without all this stuff, what do the results look like? Do we need to parse it like this?

Reply:
it's just the normal report. I just extract the needed information in the format
```
column1   label1
column2  label2
...
```